File: Source/ContactManager/Models/Domain/PackageModel.vb
'
'====================================================================================================
'
''' <summary>
''' remote method top level data structure
''' </summary>
<Serializable()>
Public Class PackageModel
    Public Property success As Boolean = False
    Public Property errorList As New List(Of PackageErrorModel)
    Public Property nodeList As New List(Of PackageNodeModel)
    Public Property profileList As List(Of PackageProfileModel)
End Class

File: Source/ContactManager/Models/Domain/PackageModel.cs
using System;
using System.Collections.Generic;

namespace Contensive.Addons.ContactManager {
    // 
    // ====================================================================================================
    // 
    /// <summary>
/// remote method top level data structure
/// </summary>
    [Serializable()]
    public class PackageModel {
        public bool success { get; set; } = false;
        public List<PackageErrorModel> errorList { get; set; } = new List<PackageErrorModel>();
        public List<PackageNodeModel> nodeList { get; set; } = new List<PackageNodeModel>();
        public List<PackageProfileModel> profileList { get; set; }
    }
}